Slim/Player/Squeezebox.pm

```diff
@@ -879,9 +879,9 @@ sub stream_s {
 	
 	# If the output channels pref is set, and the player is synced to at least 1 other active player
 	# we tell the player to only play the desired channel. If not actively synced, the player will play
-	# in stereo.
+	# in stereo, unless channels are combined
 	if ( my $outputChannels = $prefs->client($client)->get('outputChannels') ) {
-		if ( $client->isSynced(1) ) { # use active player count
+		if ( $outputChannels == 0x03 || $client->isSynced(1) ) { # use active player count or this is L+R
 			$flags |= ($outputChannels & 0x03) << 2;
```

Comment:
I need a little help here. Are you basically setting the flags to play left AND right only?

Reply:
Yes, the request is to play L+R combined to build a mono speaker. As said, not the most important feature, I agree, but was discussed recently on the forum and I thought about that to built a single speaker device.

Reply:
But do we know what Touch/Radio do if they receive both flags? As I understand it by selecting the "combined" channel the bits for both left only and right only are set. I don't know whether there's a well defined behavior in the two players.

Reply:
If I'm reading this right, all jive based players will copy the LEFT channel to RIGHT if both L+R are set in output_channels.
https://github.com/ralph-irving/squeezeos-squeezeplay/blob/public/8.0/src/squeezeplay/src/audio/decode/decode_output.c#L354

Reply:
I've tried on my Radio and it played normally, consistent with what @ralph-irving comment about the code. Having said that, this option is "passive"/not enabled anyway by default